byzcoin/internal/parse.go

```diff
@@ -0,0 +1,139 @@
+package internal
```

Comment:
What is this package for, and why? bcadmin already has a mysterious dumping ground of random util crap called bcadmin/lib. Not too keen on a second random util/crap package...